src/api/app/views/webui2/webui/kiwi/images/_details_overview.html.haml

```diff
@@ -0,0 +1,18 @@
+%h5.card-header Details
+.card-body
+  %p
+    %strong Author
+  %p
+    = @author
+    = "(#{@contact})" if @contact.present?
+  %p
+    %strong Software
+  %p
+    - if @image.use_project_repositories
+      This image uses the repositories set in the project
+    - else
+      = @repositories_count
+      repositories selected
+  %p
+    = @packages_count
+    package/s selected
```

Comment:
Check out [pluralize](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize)! It's great. You can use it here:-)